features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Operations/Operation.jsx

```diff
@@ -49,7 +49,7 @@ const styles = theme => ({
         width: 400,
     },
     dropDown: {
-        width: 100,
+        width: 90,
```

Comment:
shall we specify width relative to `theme.spacing.unit` rather than using absolute width?

Reply:
Resolved